src/components/dev-hub/podcast-player/player-contols.js

```diff
@@ -0,0 +1,107 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+
+import Button from '~components/dev-hub/button';
+import {
+    FastForwardIcon,
+    PauseIcon,
+    PlayIcon,
+    RewindIcon,
+} from '~components/dev-hub/icons/audio-player';
+
+import { screenSize, size } from '~components/dev-hub/theme';
+
+const buttonStyles = css`
+    margin-right: 12px;
+    padding: ${size.tiny};
+    &:active {
+        filter: brightness(60%);
+    }
+`;
+
+const mainButtonHover = css`
+    svg {
+        transform: scale(1.25);
+        circle {
+            stroke-width: 3;
+            r: 38;
+        }
+        path {
+            stroke-width: 3;
+        }
+    }
+`;
+
+export const ControlsContainer = styled('div')`
+    align-items: center;
+    display: flex;
+    justify-content: center;
+`;
+
+const StyledTimeControlButton = styled(Button)`
+    ${buttonStyles},
+    &:hover {
+        svg {
+            path:first-of-type {
+                transform: scale(1.15);
+            }
+        }
+    }
+`;
+
+const StyledMainButton = styled(Button)`
+    ${buttonStyles},
+    &:hover {
+        ${mainButtonHover}
+    }
+
+    @media ${screenSize.upToSmall} {
+        align-items: center;
+        display: inline-flex;
+        height: 100px;
+        justify-content: center;
+        width: 110px;
```

Comment:
For the fixed px values can we move them into constants to better describe what they are?